### PR TITLE
Improves the documentation for nuget pack -action

### PR DESCRIPTION
**Pull request by koskila**
I was really struggling to get the nuget package versioning to work as I wanted - After a lot of trial and error, I got the process to work, and used my experiences to try and make the documentation a bit more comprehensive.

Hope it's useful!

### CODE REVIEW

File: docs/pipelines/tasks/package/nuget.md [hunk 1]
```diff
@@ -89,9 +89,35 @@ If your code depends on NuGet packages, make sure to add this step before your [
...
+  inputs:
+    command: pack
+    versioningScheme: byPrereleaseNumber
+    majorVersion: '1'
+    minorVersion: '0'
+    patchVersion: '$(Patch)'
+    packTimezone: utc
+``` 
+
+For **byEnvVar**, the version will be set as whatever environment variable, e.g. `MyVersion` (no **$**, just the environment variable name), you provide. Make sure the environment variable is set to a proper SemVer e.g. `1.2.3` or `1.2.3-beta1`. If you select this option, you also need to set the environment variable, like in the example below.
```
Comment (chasewilson) [inline docs/pipelines/tasks/package/nuget.md]: Would you be able to update to match the env vars in the text and the example?

**Comment by chasewilson** [comment]
@koskila This is great work and the contribution is greatly appreciated! There's just one small update but I'll keep an eye on it and merge it ASAP :)

**Comment by PRMerger9** [comment]
@koskila : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Izzmo** [comment]
@koskila thanks for this update.. hopefully it gets merged! This documentation as a whole is lacking, so you are helping out a lot.

**Comment by ktoliver** [comment]
The updates in the pull request appear to be in master, so we'll close the PR.

**Pull request closed without merging**